tests/src/test/scala/system/basic/WskSdkTests.scala

```diff
@@ -89,23 +90,40 @@ class WskSdkTests
         FileUtils.deleteDirectory(dir)
     }
 
-    it should "install the bash auto-completion bash script" in {
-        // Use a temp dir for testing to not disturb user's local folder
+    it should "print bash command completion script to STDOUT" in {
+        val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+        val msg = "bash completion for wsk"    // Subject to change, dependent on Cobra script
+
+        val stdout = wsk.cli(Seq("sdk", "install", "bashauto") ++ wskprops.overrides ++ auth, expectedExitCode = SUCCESS_EXIT).stdout
+        stdout should include(msg)
+    }
+
+    it should "check if .bashrc exists and append it with the bash auto completion command when using --bashrc" in {
+        val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+        val homeDir = System.getProperty("user.home")
+        val bashrc = new File(homeDir, ".bashrc")
         val dir = File.createTempFile("wskinstall", ".tmp")
         dir.delete()
         dir.mkdir() should be(true)
 
-        val scriptfilename = "wsk_cli_bash_completion.sh"
-        var scriptfile = new File(dir.getPath(), scriptfilename)
+        val backUp = File.createTempFile("backup",".tmp", dir)
+        backUp.exists() should be(true)
```

Comment:
Want to move the creation of temporary files inside the `if` clause and have a `try/finally` in there to clean up?